cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/stream/FileStreamAdmin.java

```diff
@@ -96,12 +98,12 @@ public void setExploreFacade(ExploreFacade exploreFacade) {
   }
 
   @Override
-  public void dropAll() throws Exception {
+  public void dropAllInNamespace(Id.Namespace namespace) throws Exception {
```

Comment:
Messy behavior comes about when null is passed in.
Instead of checking for null at each method in the pipeline of method calls, I figure its best to ensure it at the very top (handler-level).